### PR TITLE
fix: give link on home tab now opens in external window

### PR DESCRIPTION
**Pull request by nlewis84**
@redreceipt Are you okay with this change? 

https://user-images.githubusercontent.com/72768221/170521368-3c316cb8-cb6e-44c9-8f2c-bb61e0161f3d.mov

### CODE REVIEW

**Comment by redreceipt** [comment]
I think the fox is simpler than that. Just remove the Android bit and #external links will always open in Chrome. That's what they want right?

**Comment by nlewis84** [comment]
That's what we want, but canOpenUrl returns false for that url because it is https. I've tried adding that to the the intent-filter, but no luck there. Also, tried something along these lines  `<data android:scheme="https" android:host="*" />` but no luck there (there are some documented issues for RN having issues with this and Android 10/11)

**Comment by redreceipt** [comment]
Wait I'm confused, react native can't open https links on Android? That seems hard to believe

**Comment by redreceipt** [comment]
Screen Shot 2022-05-26 at 2 03 19 PM

**Comment by redreceipt** [comment]
you've been through all this?

Screen Shot 2022-05-26 at 2 04 40 PM

**Comment by redreceipt** [comment]
maybe you just need to add the intent like that last block shows?

**Comment by redreceipt** [comment]
reason it's important is if they decide to use a different #external link for something else that isn't pushpay. it won't work at all on android given the current logic.

**Comment by nlewis84** [comment]
Could actually just reuse the logic you put in and just flip it and put it on the `canWeOpenUrl` bit.

File: apolloschurchapp/src/theme/theme.js [hunk 1]
```diff
@@ -25,16 +25,16 @@ import fontStack from './fontStack';
...
-    if (
-      url.startsWith('http') &&
-      (!url.includes('#external') || Platform.OS === 'android')
-    ) {
+    if (url.startsWith('http') && !url.includes('#external')) {
       // safe enough to use InAppBrowser
       return InAppBrowser.open(url);
     }
 
     const canWeOpenUrl = await Linking.canOpenURL(url);
```
Comment (redreceipt) [inline apolloschurchapp/src/theme/theme.js:33]:
```suggestion
    const canWeOpenUrl = (await Linking.canOpenURL(url)) || (url.startsWith('https') && Platform.OS === 'android');
```

Comment (redreceipt) [reply apolloschurchapp/src/theme/theme.js:33]: this at least captures the exact bug we can't seem to fix

**Comment by nlewis84** [comment]
That works.
https://user-images.githubusercontent.com/72768221/170565430-2cb2f1c1-a105-4d42-a1db-e1930f578508.mov

**Review by redreceipt — APPROVED** [review]
sorry for being nitpicky. i have PTSD from breaking LCBC custom features

**Pull request merged**